readthedocs/subscriptions/tests/test_event_handlers.py

```diff
@@ -283,6 +283,7 @@ def test_subscription_canceled_trial_subscription(self, notification_send):
             },
         )
         event_handlers.subscription_canceled(event)
+        event_handlers.subscription_updated_event(event)
```

Comment:
We should have an assert related to this call, doesn't look like we are testing anything here. Same for the other test without an assert.

Reply:
Yes, we are not testing anything in particular here. I added them because _they are executed_ when the webhook "subscription canceled" arrives and not calling them from the tests could have unexpected behaviors otherwise.